meta/manual_xenobladechroniclesv2_xanderoni_skipperki1.toml

```diff
@@ -0,0 +1,95 @@
+[["test.general.test_options.TestOptions.test_pickle_dumps (game='Manual_XenobladeChroniclesv2_Xanderoni_skipperki1', option='goal')"]]
```

Comment:
Can you remove those, they're outdated anyway since that test name has changed? I'll generate them through CI automatically